uber/site_sections/summary.py

```diff
@@ -175,12 +175,115 @@ def food_eligible(self, session):
         }
         return render('summary/food_eligible.xml', {'attendees': eligible})
 
+    def csv_import(self, message='', all_instances=None, form_link=None):
+
+        return {
+            'message': message,
+            'tables': sorted([model.__name__.lower() for model in Session.all_models()]),
```

Comment:
This isn't correct because our model-name-to-table-name conversion puts underscores between words, e.g.

```python
>>> TabletopCheckout.__tablename__
'tabletop_checkout'
```

As per my comment above, a better way to do this would be to have this be something like

```
'models': sorted(model.__name__ for model in Session.all_models())
```

then you can use ``Session.resolve_model(model_name)`` to get the model based on the class name.  (This is how the automatic CRUD API exposed by Sideboard works, which is why the ``resolve_model`` method exists in the first place.)